File: Leetcode-one-by-one/src/Find Unique Binary String.swift
class Solution {
    func findDifferentBinaryString(_ nums: [String]) -> String {
        let n = nums.count
        var result = ""

        for i in 0..<n {
            let index = nums[i].index(nums[i].startIndex, offsetBy: i)
            let bit = nums[i][index]
            result.append(bit == "0" ? "1" : "0")
        }

        return result
    }
}

File: Leetcode-one-by-one/src/Find Unique Binary String.go
package main

func findDifferentBinaryString(nums []string) string {
    n := len(nums)
    result := make([]byte, n)

    for i := 0; i < n; i++ {
        if nums[i][i] == '0' {
            result[i] = '1'
        } else {
            result[i] = '0'
        }
    }

    return string(result)
}
